Update Page Accueil

arihuss opened the pull request:
- Rajout de bouton «devenir membre» a la nav bar
- Rajout de la FAQ
- Rajouter une image a l'accueil
- Rajouter section evenements
- Update nav bar et rendre logo clickable

sarahhannets committed:
add link logo

arihuss committed:
mini update navbar

se rappeler de
-supprimer contact
-rajouter evenements
-embellir devenir membre (rajouter lien aussi)

arihuss committed:
Revert "mini update navbar"

This reverts commit 7a2b997bc683bce08bd57cc374508b9c0462dc1b.

arihuss committed:
change home, rajouter photo, add hover

sarahhannets committed:
avancement html et css homepage

-ajout nouvelle sections (projets, faq etc)
-ajout bouton AMC
-ajout css pour chaque projet (class project)

arihuss committed:
rajouter bouton devenir membre

arihuss committed:
FAQ, Futurs projets

-Rajouter FAQ
-Rajouter futurs projets
-Rajouter animation pour logo home

sarahhannets committed:
texte dans FAQ

sarahhannets committed:
index

-ajout page evenements
-ajout evenements dans nav bar
-enlerver copyright
-ajout des deux icones footer (mail et location)

arihuss committed:
fix faq

changer le code de faq pour que ca soit plus optimal avec google

arihuss committed:
a venir page evenement

afficher un message «a venir» lorsqu'on ouvre la page evenement pour le moment

arihuss committed:
Enlever tout ce qui est projets pour le moment

Va etre remis quand un bon design pour projet va etre trouver plus tard

LouisPhilippeHeon commented on index.html:
```diff
@@ -20,50 +21,108 @@
+        <div class="container">
+          <h3>FAQ</h3>
+          <div class="gradient-border"></div>
+          <div class="section collapsible"><p class="title">Je suis un debutant, est-il possible pour moi de joindre le  club ?</p>
```
Pour des raisons d'acessiblité, il ne faut jamais utiliser un <p> comme titre de section. Ça devrait être des h4

LouisPhilippeHeon commented on css/styles.css:
```diff
@@ -314,4 +355,18 @@ footer {
+}
+
+
+#btn-membre{
+    display: inline-block;
+    background-color: #6571A7;
+    border: 2px solid white;
+    border-radius: 10px;
+    padding: 0rem 0.4rem;
+   
+}
+
+#btn-membre:hover{
+    background-color: #535e91;
 }
```
il faudrait mettre le css pour btn membre à l'intérieur de la section pour le header
header {
btn membre
}

sarahhannets committed:
formatage

sarahhannets committed:
formatage

sarahhannets committed:
espace insecable

sarahhannets committed:
formatage

-btn-devenir membre dans header
-formatage
-enlever le lien du logo
-changer le lien de devenir membre
-class =faq minuscule
-rajouter index.css #faq {collapsible...

arihuss committed:
fix nav bar, responsiveness, animation logo

- l'accueil est responsive sur son cell mtn, pas de overflow ou de bugs
- animation plus smooth pour logo
- devenir membre n'a pas de hover sur la nav-bar

arihuss committed:
formatage

arihuss committed:
fix faq

fix bug que la faq avait un padding qui n'etait pas clickable

arihuss committed:
mini fix formatage

LouisPhilippeHeon commented on components/footer.html:
```diff
@@ -25,5 +31,4 @@ <h2 id="footer-contact">Contact</h2>
     <h1>App|ETS</h1>
     <img src="images/applets.svg" alt="">
   </div>
-  <p>© 2025 App|ETS</p>
```
maintenant que le copyright est enlevé il faut aussi enlever du css qui venait avec! ligne 176 et 294-309 de styles.css

LouisPhilippeHeon commented on components/footer.html:
```diff
@@ -25,5 +31,4 @@ <h2 id="footer-contact">Contact</h2>
     <h1>App|ETS</h1>
```
maintenant que on a plus le copy right on peut enlever un div
```
<div id=applets>
  <h1>
  <img>
</div>
```

LouisPhilippeHeon commented on css/home.css:
```diff
@@ -2,37 +2,76 @@ main {
+        }
+    }
+
+    #logo-home {
```
mettre à l'intérieur de la section hero

LouisPhilippeHeon commented on css/home.css:
```diff
@@ -57,4 +96,61 @@ main {
+        transform: rotate(-90deg);
+    }
+
+    .collapsible:nth-child(3) {
```
plutôt que faire ça, on devrait mettre tous les éléments identiques en dedans d'un div pour marquer le fait qu'ils sont groupés et sélectionner le premier

LouisPhilippeHeon commented on index.html:
```diff
@@ -19,51 +20,109 @@
-          <li>Promouvoir le développement d’applications mobiles.</li>
-        </ul>
+      
+
```
un retour de ligne de trop

LouisPhilippeHeon commented on js/home.js:
```diff
@@ -21,4 +21,36 @@ const Typewriter = function (iterationSpeed, destinationElement, inputText) {
+
+
+
+
```
retirer les lignes 51 à 56

sarahhannets committed:
formatage

arihuss committed:
corrections

- rajouter logo home dans hero
- changer child collapsible

LouisPhilippeHeon commented on css/styles.css:
```diff
@@ -221,8 +231,24 @@ header {
         }
     }
+
+    #btn-membre {
```
renommer a become-member-btn pour suivre le nommage en anglais

LouisPhilippeHeon commented on index.html:
```diff
@@ -19,44 +20,139 @@
+              <h4 class="title">Puis-je rejoindre le club en cours d’année&nbsp;?</h4>
+              <div hidden="until-found" class="content">
+                <p>Oui&nbsp;! Vous êtes le bienvenu à tout moment. Notre club est toujours ouvert aux nouveaux membres
+                  et
```
corriger le formatage

LouisPhilippeHeon commented on index.html:
```diff
@@ -19,44 +20,139 @@
+                  d’une communauté étudiante dynamique.
+                  En intégrant le club, vous aurez l’occasion de développer vos compétences en programmation mobile
+                  grâce
+                  à des ateliers et des projets,
```
grâce ne devrait pas être tout seul sur sa ligne, tout comme partie

sarahhannets committed:
fix formatage

sarahhannets reviewed (commented):
Formatge
-btn-membre
-faq
